Maple2.File.Parser/Xml/AI/Condition.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Collections.Generic;
+using System.Xml.Serialization;
+
+namespace Maple2.File.Parser.Xml.AI;
+
+public enum ConditionTargetState {
```

Comment:
There's a separate directory for enums: `Maple2.File.Parser.Enum`

Not really against this style, but I would prefer to move them there and keep things consistent.